### PR TITLE
Ruby group capstone - Catalog of my things

### PR DESCRIPTION
**Pull request by calebchris000**
## Highlights
The following requirements were implemented:
- Create the Book Class and the Label class
- Create the MusicAlbum class inherited from the item class.
- Create the Game class and Author class.
- Add the console feature to interact with the application
- Create the SQL database to persist the data of the application.
- Create tests for all the classes to ensure their expected behaviors.
- Save the data of the application to JSON files.

[Project video presentation.](https://drive.google.com/file/d/1hm14XvJYWAgZqcWq_0CvYOlGhMU6QWCJ/view?usp=share_link)

#### Cheers! 👏

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
+-- Add your create table query as per instruction.
```
Comment (gemmen29) [inline schema.sql:1]: - [ ] Kindly add indexes for all foreign keys to speed up the retrieval of data from tables.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (shubhambhoyar077) [reply schema.sql:1]: Hi @gemmen29 ,
Thanks for review. :smile: 
We have added indexes for foreign key.

**Review by amejid — APPROVED** [review]
Hi team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Linter checks passing :heavy_check_mark: 
- Correct workflow :heavy_check_mark: 
- Indexes added on foreign keys :heavy_check_mark: 
- Good test coverage :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by shubhambhoyar077** [comment]
Hi @amejid ,
Thanks for Approval. :smile:

**Pull request merged**